jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -37,7 +32,26 @@ toc::[]
 
 ## Migration
 
+WARNING: Migrating to Server v3.x will shut down your v2.19.x application. Your v2.19.x application will not be started back up,
+although you may manually start it back up using the administrative console.
+
+WARNING: Starting the migration process will cause downtime. It is recommended you schedule a maintenance window.
+
+WARNING: Running Server 2.19 and Server 3.0 at the same time can cause issues to your 2.19 build data. Server 2.19 should NOT be restarted if Server 3.0 is running.
```

Comment:
WARNING: Running Server 2.19 and Server 3.0 at the same time can cause issues to your build data in both instances. Consult customer support before restarting Server 2.19 if Server 3.0 is running.

Reply:
I see I am quite late.  This is a small clarification that contaminated data from running both instances at the same time can flow both ways.  I will defer to you @annabaker if this needs to be included at all